network/load-balancer/concepts.mdx

```diff
@@ -205,14 +259,22 @@ Scaleway Load Balancers can use SNI to define [routes](#routes) from a frontend
 A sticky session enables the Load Balancer to bind a user's session to a specific server in the backend pool. This ensures that all subsequent sessions from the user are sent to the same backend server while there is at least one active session. Sticky sessions can be **cookie based** or **table based**. 
 
 - **Cookie-based** stickiness uses a HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
```

Comment:
```suggestion
- **Cookie-based** stickiness uses an HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
```